x-pack/plugins/alerts/server/alerts_client/alerts_client.ts

```diff
@@ -334,10 +340,14 @@ export class AlertsClient {
       });
       createdAlert.attributes.scheduledTaskId = scheduledTask.id;
     }
-    return this.getAlertFromRaw(createdAlert.id, createdAlert.attributes, references);
+    return this.getAlertFromRaw<Params>(createdAlert.id, createdAlert.attributes, references);
   }
 
-  public async get({ id }: { id: string }): Promise<SanitizedAlert> {
+  public async get<Params extends AlertTypeParams = never>({
```

Comment:
Using `never` here felt right - we don't force them to specify the type, _unless_ they want to use the field, in which case they'll have to be explicit.